app/lib/route-helpers.js

```diff
@@ -83,10 +84,22 @@ const isUrlEndingFromList = (url, endingsToExclude) => {
   return found
 }
 
+const getPoliceForceName = async (request, user) => {
+  let policeForceName = getFromSession(request, constants.keys.policeForceDisplayName)
+
+  if (!policeForceName) {
+    policeForceName = await getPoliceForceDisplayName(user)
+    setInSession(request, constants.keys.policeForceDisplayName, policeForceName)
+  }
+
+  return policeForceName
+}
```

Comment:
not one for now due to the rush, but we could add this to the user details so it's always available